### ISSUE TITLE
build from source

### ISSUE THREAD

**Issue by adalisan** [ethz-asl/robust_point_cloud_registration#7]
I am trying to build these project from source. There are too many dependencies that I cannot keep track of  and I don't have root privilages on this machine.
While I installed catkin_tools, I don't have a ROS distrubition which seems like it's necessary?
What's an easier way to build from source?

**Comment by hitimo** [comment]
Yes, currently the code is set up as a ROS node since we used it to communicate between robots. So currently a ROS installation (tested for indigo and kinetic) is required which in turn requires root privileges. The wstool then automatically handles the dependencies which are listed [here](https://github.com/ethz-asl/robust_point_cloud_registration/blob/master/install/dependencies.rosinstall).

I agree, one should have the option to install with or without ROS. I opened an issue: #8

**Comment by la9881275** [comment]
After running “catkin init”, the output is "catkin: command not found".
I do not clearly understand how to compile your package.
